### ISSUE TITLE
[Accessibility]A11y_VSCode_Feedback_Keyboard:Controls are not accessible by keyboard.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51872]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**
Keyboard,MAS2.1.1
**Repro Steps:**
1. Launch VS Code.
2.Navigate to Status Bar -->weet Feedback" button.
3. Check the keyboard navigation.

**Actual:**
"Tweet Feedback" button is not accessible by keyboard.it is only accessible through mouse.

**Expected:**

"Tweet Feedback" button should accessible through keyboard.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The keyboard user will not be able to access the control.
**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

Does this issue occur when all extensions are disabled?: Yes/No
**Attachment For Reference**

[[Accessibility]A11y_VSCode_Feedback_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2101578/Accessibility.A11y_VSCode_Feedback_Keyboard.pptx)

**Comment by joaomoreno** [comment]
cc @bpasero Is the status bar accessible at all?

**Comment by bpasero** [comment]
@joaomoreno every entry has a related global command, so you do not have to execute something from down there imho via keyboard.

**Comment by joaomoreno** [comment]
@AccessibilityTestingTeam-TCS Would making a global command for opening the Feedback widget solve this?
